src/index.html

```diff
@@ -7,12 +8,61 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The stylesheet link is pointing to a '.scss' file. Ensure that this file is compiled to a '.css' file, as browsers cannot directly interpret Sass files. Update the href attribute to point to the compiled CSS file.